playbooks/install.yml

```diff
@@ -0,0 +1,41 @@
+- name: Generate Facts
+  hosts: all
+  strategy: linear
+  run_once: true
+  tasks:
+  - name: Preflight checks
+    delegate_to: localhost
+    block:
+    - name: Check that there is one and only one bluechi_controller
+      ansible.builtin.assert:
+        that: groups['bluechi_controller'] | length == 1
+        fail_msg: There must be one and only one bluechi_controller
+
+  - name: Calculate facts
+    delegate_to: localhost
+    block:
+    - name: Set the Controller address
+      ansible.builtin.set_fact:
+        agent_controller_host: "{{ hostvars[groups['bluechi_controller'][0]]['ansible_facts']['default_ipv4']['address'] }}"
```

Comment:
Hmm, we should file an issue to support also IPv6 only networks in future

Reply:
Done #2